google/cloud/testing_util/integration_test.cc

```diff
@@ -0,0 +1,30 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "google/cloud/testing_util/integration_test.h"
+#include "google/cloud/log.h"
+
+namespace google {
+namespace cloud {
+inline namespace GOOGLE_CLOUD_CPP_NS {
+namespace testing_util {
+
+void IntegrationTest::TearDown() {
```

Comment:
If we're going to be careful to call `IntegrationTest::Setup()` from the sub-class `Setup()` functions (which will just resolve to `::testing::Test::Setup()`), then we should probably call `::testing::Test::TearDown()` explicitly before this `TearDown()` override returns.

Reply:
Done.